study-builder/studies/pancan/snippets/common/cancer-picklist-groups-additional.conf

```diff
@@ -0,0 +1,323 @@
+{
+  "stableId": "CANCER_GROUP_ADDITIONAL_ADDITIONAL_CANCERS",
+  "nameTemplate": {
+    "templateType": "HTML",
+    "templateText": "$prompt",
+    "variables": [
+      {
+        "name": "prompt",
+        "translations": [
+          { "language": "en", "text": ${i18n.en.cancer.cancer_group_additional_additional_cancers} },
+          { "language": "es", "text": ${i18n.es.cancer.cancer_group_additional_additional_cancers} },
+        ]
+      }
+    ]
+  }
+  "options": [
+
+    {
+      "stableId": "CANCER_ADDITIONAL_DF_DESMOID_FIBROSIS",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$cancer_additional_df_desmoid_fibrosis",
+        "variables": [
+          {
+            "name": "cancer_additional_df_desmoid_fibrosis",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.cancer.cancer_additional_df_desmoid_fibrosis} },
+              { "language": "es", "text": ${i18n.es.cancer.cancer_additional_df_desmoid_fibrosis} },
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "CANCER_ADDITIONAL_DSRCT",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$cancer_additional_dsrct",
+        "variables": [
+          {
+            "name": "cancer_additional_dsrct",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.cancer.cancer_additional_dsrct} },
+              { "language": "es", "text": ${i18n.es.cancer.cancer_additional_dsrct} },
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "CANCER_ADDITIONAL_CUP__UNKNOWN_PRIMARY",
```

Comment:
Can we _not_ use double-underscore? Keep in mind that the option stable ids will show up in data exports. And study staff looks at data exports and do data analysis on them. Having double-underscore might confuse study staff.

Reply:
@yufengwng sure. The only thing where this underscore needed - to separate a "subgroup" (options which are synonyms, were defined on the same line of Excel).
For example below options are defined on the same line of the Excel doc:
```
  "cancer_skin_adnexal__sat_skin_adnexal": "Skin adnexal tumors (SAT)",
  "cancer_skin_adnexal__cutaneous_tumor": "Cutaneous adnexal tumor",
  "cancer_skin_adnexal__sebaceous_carc": "Sebaceous carcinoma",
```
If we replace double underscore with underscore then it becomes more problematic to identify a subgroup of synonyms (or similar cancers - I don't know which rule they follow when defined cancers on one line). But now (looking at a stableId) it is clear that this group are synonyms.

Reply:
Removed double underscore (used only single underscores). Instead - added prefix "S_" before a name of a subgroup - in order to be possible to identify a subgroup.

Reply:
I'm not sure if study staff will want to see a `S_`. I think I will recommend removing `S_`.

Reply:
evgeni: I believe stabelDs will look cleaner without C_ and S_ but lets go ahead and merge for now.
We can revisit next week and do more cleanup time permitting.

Reply:
Ok, in that case I agree with you about removing `C_` and `S_` (I didn't think about study staff usage of this data) - it should be removed. 
Sampath advised to remove it in next iteration, as they want to merge it today and then apply.